integration-tests/cli/two-source/test.test.js

```diff
@@ -0,0 +1,10 @@
+const test = require('ava');
+const {exec} = require('../../setup');
+
+test.failing(
```

Comment:
So, again, should this be `test.failing`? (Sorry if I missed an answer somewhere.)
IMHO we can use `test` for things that should work and `test.failing` for things that should fail (compiler errors etc.).
This seems like something that should work (but perhaps doesn't yet, I don't know and I'm AFK :D ) so I'd expect `test`

Reply:
See https://github.com/elm-in-elm/compiler/pull/105#issuecomment-652056367.

We don't want failing tests on master do we? I am definitely happy to drop this test from the PR. I added it when I thought I could also add a fix for it.

Reply:
Ah, thanks, apparently I'm blind :)
Well I'd rather mark it as "should not fail but fails".

> Failing tests are fine for now :) Perhaps that will change as we stabilize, but for now they're OK.

So I guess either 
1. mark it as `test` and add a fix so that it passes
2. mark it as `test` and let it be :x: in master (we'll fix it later)
3. remove the test (perhaps add a TODO somewhere?)

But I don't like pretending it _should_ fail.

Reply:
My understanding (see [ava docs](https://github.com/avajs/ava/blob/f72fab40160a413f4d7ab0208979fe3e721198ea/docs/01-writing-tests.md#failing-tests)) is that the `.failing` does _not_ mark a test as "meant to fail" but instead "currently failing and needs to be fixed". In this way `.failing` works the same as `.skip` but ava checks that the test does actually fail and so you get prompted to remove the `.failing` if you fix the issue.

Reply:
Option 1. is definitely preferable to me than option 2. otherwise

Reply:
Hm, then we should change our usages of `test.failing` for something else, because they all should fail. I'm gonna read the docs...

Reply:
I guess the (expected) compiler failures should be somehow test successes. We'll need to rethink this a bit.

Anyways, with the newly acquired knowledge, `test.failing` is fine for the "not yet implemented" test. :+1:

Reply:
The only other `.failing` test is currently https://github.com/elm-in-elm/compiler/search?q=failing which is also a  "not yet implemented" test so I think we may be okay :)

Reply:
My bad, I'm in the type-annotations branch in my head... I was using the integration tests wrong there. Added about 7 failing tests that are implemented fully and supposed to fail. :see_no_evil:

Reply:
I put some (hopefully helpful) comments over in that PR :)